actionview/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   `button_to` now accepts a `button_tag:` option, to override the `button_to_generates_button_tag` config
+
+    ```ruby
+    # With config.action_view.button_to_generates_button_tag = false
+
+    button_to("Save", "http://www.example.com", button_tag: true) # renders <button>
```

Comment:
Since the new default is to render a `<button>`, what do you think about changing the keyword argument to be `<input>`-focused? For example, `input_tag: true` or `input: true`?